framer/Animation.coffee

```diff
@@ -190,7 +190,11 @@ class exports.Animation extends EventEmitter
 	_updateValue: (value) =>
 
 		for k, v of @_stateB
-			@_target[k] = Utils.mapRange(value, 0, 1, @_stateA[k], @_stateB[k])
+
```

Comment:
General issues:

- Make sure you consistently use `'` or `"`. We use `"` in the rest of the source.